addons/website/controllers/main.py

```diff
@@ -176,7 +176,7 @@ def sitemap_xml_index(self, **kwargs):
 
         def create_sitemap(url, content):
             return Attachment.create({
-                'datas': base64.b64encode(content),
+                'datas': base64.b64encode(content.encode('utf-8')),
```

Comment:
simpler, there is now a `raw` field that allows writing without having to encode
(also "utf-8" is the default encode in P3)
```suggestion
                'raw': content,
```